mantidproject/mantid
# Add nx can sas to isis sans gui

Fixes #17583

The new file format NXcanSAS will be the new standard for ISIS SANS reduction results. We want to make available with this PR.

**To test:**


The relevant data can be found here: \\olympic\babylon5\Scratch\Anton\Testing\17583_add_nx_can_sas_option

Note that the folder contains a Windows installer

1. Open the ISIS SANS GUI
2. Set the instrument to SANS2D
3. Load the user file: USER_SANS2D_154E_2p4_4m_M3_Xpress_8mm_SampleChanger.txt
4. Set the workspaces to:
sample_scatter: 34484
sample_trans: 34502
sample_direct: 34461
can_scatter: 34481
can_trans: 34505
can_direct: 34461

5. Press load
6. Press 1DReduce
7. Once the reduction has completed, in the save section, only select `NxCanSAS` and press `SaveResult`
  * Confirm that the workspace is saved out
8. Load the file into mantid using `LoadNXcanSAS`
  * Confirm that it can be loaded
9. Delete the saved file
10. Press save other
11. Select only `NXcanSAS` and set the workspace to `34484rear_1D_1.75_16.5` which is the reduced workspace
12. Press save
  * Confirm that the workspace was saved out
13. Load the file into mantid using `LoadNXcanSAS`
  * Confirm that it can be loaded
  * Load the 



# Release notes

For release notes, see [here](https://github.com/mantidproject/mantid/pull/17928/commits/bd11db67a49e01360ea969036d87a666f4bcd387)

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


- Does everything look good? Comment with the ship it emoji but don't merge. A member of `@mantidproject/gatekeepers` will take care of it.
